lib/doh.c

```diff
@@ -42,9 +42,17 @@
 #include "curl_printf.h"
 #include "curl_memory.h"
 #include "memdebug.h"
+#ifdef USE_HTTPSRR
+# include "escape.h"
+#endif
 
 #define DNS_CLASS_IN 0x01
 
+#ifdef USE_HTTPSRR
+/* local_print_buf truncates if the hex string will be more than this */
+# define LOCAL_PB_HEXMAX 400
+#endif
```

Comment:
Same here, just for a define that is not used otherwise anyway I don't think ifdef is needed but instead just makes it harder to read.

Reply:
done